packages/SwingSet/src/initializeSwingset.js

```diff
@@ -205,7 +205,7 @@ export function loadSwingsetConfigFile(configPath) {
     normalizeConfigDescriptor(config.bundles, dirname, false);
     // normalizeConfigDescriptor(config.devices, dirname, true); // TODO: represent devices
     if (!config.bootstrap) {
-      throw Error(`no designated bootstrap vat in ${configPath}`);
+      assert.fail(X`no designated bootstrap vat in ${configPath}`);
     } else if (!config.vats[config.bootstrap]) {
       throw Error(
```

Comment:
We could transform this second `throw` as well, or fill the body of the `else {` with `assert(config.vats[config.bootstrap], ..text)`.

Reply:
Making this into two sequential `assert` tests. Reads much cleaner.